packages/okta-auth-js/lib/browser/browser.js

```diff
@@ -263,6 +263,7 @@ proto.revokeAccessToken = async function revokeAccessToken(accessToken) {
   var sdk = this;
   if (!accessToken) {
     accessToken = await sdk.tokenManager.get('accessToken');
+    sdk.tokenManager.remove('accessToken');
```

Comment:
How does this interact with the lines below...if there is no access token, get one, then remove it, and then revoke it?   I'm not clear with the logic here (I trust you have it right, except for the second if(!accessToken) below, but the code doesn't tell me why you're doing this)

Reply:
The goal I want to achieve is to guarantee no revoked token is returned from `tokenManager.get` after revoking the token.

As the token.revoke method is accepting a token instead of a key, so the logic is to read token into variable, then call revoke with the read token.

Reply:
Comment the thinking, since it's not clear from the code?  Also, can't we remove the if(!accessToken) below since it can't happen?

Reply:
I think it can happen if there is no input and no accessToken in storage.